pkg/clusters/addons/kong/builder.go

```diff
@@ -36,6 +39,18 @@ func (b *Builder) WithPostgreSQL() *Builder {
 	return b
 }
 
+func (b *Builder) WithEnterprise(dbMode DBMode) *Builder {
```

Comment:
public symbols need godoc comments:

```suggestion
// WithEnterprise deploys Kong with enterprise proxy instead of the default proxy.
func (b *Builder) WithEnterprise(dbMode DBMode) *Builder {
```

Reply:
67c9bddf4a2fc7aaa96ca14ad3e1537d68fa473d